packages/common/src/components/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './FormGroupWithHelpText';
 export * from './HelpIconPopover';
 export * from './Icons';
 export * from './LoadingDots';
+export { default as LoadingSpinner } from './LoadingSpinner';
```

Comment:
no need to add it to index file, we are going to remove those index's completely

Reply:
Unfortunately, it looks like we cannot import components without linting/compliation errors with the following from `@kubev2v/common`:

```
import LoadingSpinner from '@kubev2v/common/src/components/LoadingSpinner';
import LoadingSpinner from '@kubev2v/common/dist/components/LoadingSpinner';
```

Both of these lead to issues, and to solve this issue is to solve a much broader one which would deal with the exporting of components collectively from that repo within this monorepo.

So unless there's something else we can do in the meantime for this, I think adding this export to the existing index is the least invasive change without changing the scope of this issue right now.